api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/loader/request/LoaderFixedFeeDto.java

```diff
@@ -14,8 +14,8 @@
 @EqualsAndHashCode(callSuper = true)
 public class LoaderFixedFeeDto extends LoaderFeeDto {
 
-    public LoaderFixedFeeDto(String code, String newCode, LoaderFeeVersionDto version, String jurisdiction1, String jurisdiction2, String service, String channel, String event, String applicantType, boolean unspecifiedClaimAmount, String keyword) {
-        super(code, newCode, version, jurisdiction1, jurisdiction2, service, channel, event, applicantType, unspecifiedClaimAmount, keyword);
+    public LoaderFixedFeeDto(String code, String newCode, LoaderFeeVersionDto version, String jurisdiction1, String jurisdiction2, String service, String channel, String event, String applicantType, boolean unspecifiedClaimAmount, String keyword, String reasonForUpdate) {
+        super(code, newCode, version, jurisdiction1, jurisdiction2, service, channel, event, applicantType, unspecifiedClaimAmount, keyword, reasonForUpdate);
```

Comment:
What is the requirement to add 'reasonForUpdate' in as attribute in FeeDTO Model ?

Reply:
This also updated but forgot to push, did that along with Encode